static/js/i18n/place.json

```diff
@@ -13,15 +13,15 @@
   },
   "chart_clause:places_within_place": {
     "defaultMessage": "places within {place}",
-    "description": "Clause for places within some other place. Used for choropleth map chart titles like Unemployment rate: places within California."
+    "description": "Clause for places within some other place. Used for choropleth map chart titles like Unemployment rate: places within Tamil Nadu."
   },
   "chart_clause:placetype_near_place": {
     "defaultMessage": "{placeType} near {place}",
-    "description": "Preposition for somewhere closeby. Used for choropleth map chart titles like Unemployment rate: counties near Travis County."
+    "description": "Preposition for somewhere closeby. Used for choropleth map chart titles like Unemployment rate: counties near Travis County or Annual rainfall: towns near Taos."
```

Comment:
Preposition for somewhere close by. Used for choropleth map chart titles like \"Unemployment rate: {counties} near {Travis County}\" or \"Annual rainfall: {towns} near T{aos}\"

In this case, I think it'll be best to allow the translators to completely build the title format, so we should include the chart topic as well in this.

Reply:
Ditto for the remaining chart title strings